processor/src/test/java/org/mapstruct/ap/test/bugs/_1608/Book.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.bugs._1608;
+
+/**
+ * @author Filip Hrisafov
+ */
+public class Book {
+
+    private String isbn;
```

Comment:
should you add a boolean target accessor?

Reply:
Not sure if it is really needed, we have other tests for that